packages/expo-av/android/build.gradle

```diff
@@ -5,6 +5,11 @@ apply plugin: 'maven'
 group = 'host.exp.exponent'
 version = '9.2.0'
 
+def NODE_MODULES = "${rootDir}/.."
+if (new File("${NODE_MODULES}/node_modules").exists()) {
+  NODE_MODULES = "${NODE_MODULES}/node_modules"
+}
```

Comment:
@tsapeta will this break monorepos without symlinks?

Reply:
Why is this a problem? Do you guys have a solution for this? I only need the `node_modules/react-native` directory for the header search paths as well as the reactnativejni lib